stetho-sample/src/debug/java/com/facebook/stetho/sample/CalendarProviderDatabasePeer.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.sample;
+
+import android.annotation.TargetApi;
+import android.content.ContentResolver;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteException;
+import android.net.Uri;
+import android.os.Build;
+import android.provider.CalendarContract;
+import com.facebook.stetho.inspector.jsonrpc.JsonRpcPeer;
+import com.facebook.stetho.inspector.protocol.module.Database;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This is example of extending database domain by adding calendar provider
+ */
+public class CalendarProviderDatabasePeer extends Database.DatabasePeer {
```

Comment:
`DatabasePeer` is a strange name for this responsibility.  Prefer `DatabaseDriver` or something.  It really has little to do with the Stetho/Chrome peer that has attached and more to do with adapting the Chrome protocol to something implementable inside the context of an Android app.